packages/vscode-extension/src/webview/views/DevicesNotFoundView.tsx

```diff
@@ -9,8 +9,8 @@ import { useDevices } from "../providers/DevicesProvider";
 import { AndroidSupportedDevices, iOSSupportedDevices } from "../utilities/consts";
 import { IOSDeviceTypeInfo, IOSRuntimeInfo } from "../../common/DeviceManager";
 import { useDependencies } from "../providers/DependenciesProvider";
-import { vscode } from "../utilities/vscode";
 import { Platform, useUtils } from "../providers/UtilsProvider";
+import { mapModelToId } from "../../devices/supportedDevices";
```

Comment:
webview should only reference code from under webview and common. this way we have clear separation on what gets bundled where. If you want to use it here, it should live under common